src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationGeneratorConfiguration.java

```diff
@@ -4099,4 +4099,19 @@ public List<NotificationGenerator> internalSubmitAppealInTimeNotificationGenerat
         );
 
     }
+
+    @Bean("internalEndAppealAutomaticallyNotificationGenerator")
+    public List<NotificationGenerator> internalEndAppealAutomaticallyNotificationGenerator(
+            AdminOfficerEndAppealAutomaticallyPersonalisation adminOfficerEndAppealAutomaticallyPersonalisation,
```

Comment:
Email needs to be sent out to DET not admin. Please follow the naming convention accordingly.

Reply:
Refactored that, as well in tests and application.yml props